cfpb/consumerfinance.gov
# Update Wagtail screenshots

The Wagtail screenshots we use in the docs are in need of updating to match the new interface of Wagtail 2.10.

---

## Additions

- Added a new feature flags screenshot to account for the revamped flags UI

## Removals

- Removed a number of screenshots that seemed to be unused, including all of the screenshots showing the old mega-menu UI

## Changes

- Updated all the old Wagtail screenshots that are still being used in the docs to show the Wagtail 2.10 UI
- Updated the feature flags documentation a bit to reflect the new flow for editing a flag's conditions

## How to test this PR

1. Make sure the new screenshots look OK on the documentation pages where they're used: `/feature-flags/`, `/translation/`, and `/editing-components/`
2. Make sure none of the removed screenshots were needed or used in any docs pages (I searched the code and didn't find any spots the filenames were used, but I'm not sure if I missed anything)

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
